src/artifact_providers/__tests__/base.test.ts

```diff
@@ -0,0 +1,41 @@
+import { stringToRegexp } from '../../utils/filters';
+import { parseFilterOptions, RawFilterOptions } from '../base';
+
+describe('parseFilterOptions', () => {
+  test('empty object', () => {
+    const rawFilters: RawFilterOptions = {};
+    const parsedFilters = parseFilterOptions(rawFilters);
+    expect(parsedFilters).not.toHaveProperty('includeNames');
+    expect(parsedFilters).not.toHaveProperty('excludeNames');
+  });
+
+  test('undefined properties', () => {
```

Comment:
I think we should also have one undefined one string combination.

Moreover, you can probably use `test.each` to clean up some of these cases. This is optional but the additional test cases is something we should definitely add.